app/components/blacklight/document_component.rb

```diff
@@ -72,11 +75,13 @@ class DocumentComponent < Blacklight::Component
     # @param document_counter [Number, nil] provided by ViewComponent collection iteration
     # @param counter_offset [Number] the offset of the start of the collection counter parameter for the component to the overall result set
     # @param show [Boolean] are we showing only a single document (vs a list of search results); used for backwards-compatibility
```

Comment:
Can we add a note or example of what it takes to subclass, so we can explain the `**args` situation?